PowerEditor/installer/nsisInclude/tools.nsh

```diff
@@ -171,7 +171,7 @@ Function writeInstallInfoInRegistry
 	WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "Publisher" "Notepad++ Team"
 	WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "MajorVersion" "${VERSION_MAJOR}"
 	WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "MinorVersion" "${VERSION_MINOR}"
-	WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "UninstallString" "$INSTDIR\uninstall.exe"
+	WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "UninstallString" '"$INSTDIR\uninstall.exe"'
```

Comment:
Should this be double quoted?
Like this
![image](https://user-images.githubusercontent.com/9664141/128919475-cd7bdb7a-a83e-4540-b72c-d20053b538c7.png)

Reply:
For some reason, GitHub failed to upload my image. So here it is:
https://imgur.com/2d7Ef59

Reply:
@mere-human 
Shouldn't it be? What's your argument?

Reply:
@mere-human 
Just updated my explanation of this security flaw.

Reply:
@mere-human The single quotes are correct to make nested double quotes appear in the string which will actually go to the registry.

See this link for confirmation, originally provided by @ozone10 : https://nsis.sourceforge.io/mediawiki/index.php?title=How_can_I_use_quotes_in_a_string%3F

Reply:
Thanks @MarkusBodensee that answers my question.